examples/windows/kubernetes/configmap.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fluentbit-config
+data:
+  fluent-bit.conf: |-
+    [SERVICE]
+      Flush             5
+      Daemon            off
+      Log_Level         info
+
+    [INPUT]
+      Name              tail
+      Path              c:\\logs\\*.log

```

Comment:
I see that log file path is C:\var\log\containers\*.log. why we are using c:\\logs\\*.log here ?